src/content/blog/release-announcement-0.2.0/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+author: savitharaghunathan
+date: 2023-06-30
+description: >-
+  Konveyor v0.2.0 Release Announcement
+featured: false
+tags:
+  - Konveyor, Kubernetes, AppModernization
+title: Konveyor v0.2.0 Release Announcement
+---
+We are thrilled to announce the release of Konveyor 0.2.0! This release supports various features, like migration waves, integration with Jira,
+and other valuable minor updates.
+
+Key Highlights of Konveyor 0.2.0:
+* Migration waves
+* Jira Integration
+* Minor improvements
+
+### :sparkles:  Migration Waves:
+We are excited to introduce a significant addition to the Konveyor project, aimed at providing enhanced value throughout the migration process. 
+We recognized the need to address the planning aspect required to scale the migration process across entire application portfolios. 
+When dealing with a large-scale portfolio containing hundreds or even thousands of applications, 
+it becomes impractical to execute the migration process in one go. To tackle this challenge, 
+we have introduced the ability for Konveyor to define migration waves, allowing for an iterative and manageable approach. 
+By breaking down the portfolio into waves, we enable a more sophisticated process that can automate the calculation of these waves based on various criteria. 
```

Comment:
"we enable a more sophisticated process that **in the future we expect will** automate the calculation of these waves based on various criteria."

The automated calculation of migration waves is not there yet

Reply:
Thanks @rromannissen. I have updated the sentence with your suggestions.